src/index.js

```diff
@@ -1,6 +1,24 @@
 import './index.css';
 import logo from './img/logo.svg';
+import getShows from './modules/api.js';
+import renderShows from './modules/renderShows.js';
 
 // add logo to the header
 document.getElementById('logoWebpage').setAttribute('src', logo);
 document.getElementById('logoWebpage-footer').setAttribute('src', logo);
+const errorMessage = document.getElementById('error-message');
+
+window.addEventListener('DOMContentLoaded', async () => {
+  try {
+    const response = await getShows();
+
+    if (!response.ok) {
+      throw new Error('Failed to fetch Series');
+    }
+
+    const shows = await response.json();
```

Comment:
Hi, dear partner!! @PabloBona 
Your code works as expected. Good job! 🎉🎉🎉

**This is completely [OPTIONAL]:** 

The logic that resolves the entire promise (without the `renderShows(shows);` function call) could be encapsulated in a function and simply called within this event handler. This could improve the organization and future reuse of your code.

Keep up the excellent work! Your progress in this project is very good and valuable. 👏👏👏